opentracing/defaulttracer.go

```diff
@@ -0,0 +1,59 @@
+package opentracing
+
+var (
+	defaultOpenTracer OpenTracer = noopOpenTracer{noopTraceContextSource{}}
+)
+
+// Should be called as early as possible in main(), prior to calling the
+// `StartSpan*` (etc) global funcs below. Prior to calling `InitDefaultTracer`,
+// any Spans started via the `StartSpan*` globals are noops.
+//
+// See `NewStandardTracer(...)` to create an `OpenTracer` instance.
```

Comment:
I like it!  I would even remove that comment in line 11 - a developer enabling a tracing implementation in his application doesn't need to know about different ways of creating a tracer. If, say, `zipkin` implementation happens to be ok with standard tracer, it can expose a single method to the developer:
```Go
package zipkin

func NewTracer(...) (OpenTracer, error) {
    return NewStandardTracer(newZipkinReporter(...), newZipkinContextSource(...)), nil
}
```
So developers are never exposed to things like reporter and context source